wercker-step.yml

```diff
@@ -1,3 +1,3 @@
 name: rails-database-yml
-version: 1.0.2
```

Comment:
Could you tell me why you downgraded version?
If you will send PR to wercker's step, this might not be necessary :)